### PR TITLE
fix rack blowing up when trying to soft-restart puma because the pid …

### PR DESCRIPTION
**Pull request by grosser**
…still exists

needs https://github.com/rack/rack/issues/1159 or https://github.com/puma/puma/issues/1060 to get resolved

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -5,10 +5,10 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.cache_classes = true
```
Comment (henders) [inline config/environments/development.rb]: These 2 changes will screw up hot-reloading in dev mode right?

**Comment by grosser** [comment]
ops, need to revert ... added while testing :D

**Comment by grosser** [comment]
fixed

**Comment by henders** [comment]
Cool, 👍

**Pull request merged**